src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -211,6 +211,8 @@ public static (SyntaxToken openBrace, SyntaxToken closeBrace) GetBraces(this Syn
                     return (property.OpenBraceToken, property.CloseBraceToken);
                 case WithExpressionSyntax withExpr:
                     return (withExpr.Initializer.OpenBraceToken, withExpr.Initializer.CloseBraceToken);
+                case ImplicitObjectCreationExpressionSyntax newExpr when newExpr.Initializer != null:
+                    return (newExpr.Initializer.OpenBraceToken, newExpr.Initializer.CloseBraceToken);
```

Comment:
```suggestion
                case ImplicitObjectCreationExpressionSyntax { Initializer: { } initializer }:
                    return (initializer.OpenBraceToken, initializer.CloseBraceToken);
```

Reply:
that wont work due to a bug https://github.com/dotnet/roslyn/issues/44534

Reply:
wait, it doesnt only if we use newExpr. I can do that